src/algopy_testing/op.py

```diff
@@ -847,6 +847,62 @@ def app_opted_in(
     return app_id in account_data.opted_apps
 
 
+class _AcctParamsGet:
+    def __getattr__(
+        self, name: str
+    ) -> typing.Callable[[algopy.Account | algopy.UInt64 | int], tuple[typing.Any, bool]]:
+        def get_account_param(a: algopy.Account | algopy.UInt64 | int) -> tuple[typing.Any, bool]:
+            import algopy
+
+            from algopy_testing.context import get_test_context
+
+            context = get_test_context()
+            if not context:
+                raise ValueError(
+                    "Test context is not initialized! Use `with algopy_testing_context()` to "
+                    "access the context manager."
+                )
+
+            active_txn = context.get_active_transaction()
+            if not active_txn:
+                raise ValueError("No active transaction found to reference asset")
+
+            account_data = None
+
+            if isinstance(a, (algopy.Account)):
+                account_data = context.get_account(str(a))
+            elif isinstance(a, (algopy.UInt64 | int)):
+                try:
+                    active_txn = context.get_active_transaction()
+                    assert (
+                        active_txn is not None
+                    ), "No active transaction found to reference account"
+                    account_data = active_txn.accounts(a)
+                except KeyError:
```

Comment:
Should have two `try`/`except` blocks here as it is two different statements that can cause these errors